app/assets/javascripts/components/drafts/draft_body.es6.jsx

```diff
@@ -29,33 +29,33 @@ class DraftBody extends Component {
   // --------------------------------------------------
   // Helpers
   // --------------------------------------------------
-  send() {
-    ComposeActions.sendEmail(this.state.template);
+  renderSave() {
+    if(this.props.template.saved) {
+      return (
+        <span>{this.props.template.saved}</span>
+      );
+    }
   }
 
-  saveDraft(field) {
-    return(event) => {
-      ComposeActions.saveDraft(event.target.value);
-    };
+  send() {
+    DraftActions.sendEmail(this.state.template);
   }
 
+
   render() {
     var draft = this.props.draft;
     var template = this.props.template;
     return (
       <div style={this.styles.container}>
-        <CardInput
-          action={this.saveDraft()}
+        <DraftSubject
+          id={draft.id}
```

Comment:
Instead of passing all these individual props, why don't we just pass down the `draft` object to `DraftSubject` and `DraftContent`? We also don't need to send the `label` as a prop, since these components aren't intended to be reusable.